macOS/clean/ls.sdiff.trigger_2

```diff
@@ -0,0 +1,5 @@
+*** changed: macOS/clean/ls
```

Comment:
this file can be left in the bincapz repo.

Reply:
Oh, good catch. I tried to scrub all of the test files and missed these.

Reply:
Removed in [`ac45782` (#1)](https://github.com/chainguard-dev/bincapz-samples/pull/1/commits/ac45782ef643cbd82663812d710ad10d4076ddc5).